TaylorFramework/Modules/scissors/OffsetRange.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  OffsetRange.swift
+//  Taylor
+//
+//  Created by Alexandru Culeva on 4/12/16.
+//  Copyright © 2016 YOPESO. All rights reserved.
+//
+
+import Foundation
+
+struct OffsetRange {
+    var start: Int
+    var end: Int
+    
+    init(start: Int, end: Int) {
+        self.start = start
+        self.end = end
+    }
+}
+
+extension OffsetRange: Comparable { }
+
+func == (lhs: OffsetRange, rhs: OffsetRange) -> Bool {
```

Comment:
Coding style: there should be no space between functions(and operator declaration is a function) and their arguments list.